VideoWeb/VideoWeb.UnitTests/Controllers/ConferenceManagement/DismissParticipantTests.cs

```diff
@@ -137,23 +137,74 @@ public async Task should_return_accepted_when_participant_is_witness_and_judge_i
                         r.ParticipantId == witness.Id && r.TransferType == TransferType.Dismiss)), Times.Once);
         }
 
+        [Test]
+        [TestCase(Role.QuickLinkObserver, "Observer")]
+        [TestCase(Role.QuickLinkParticipant, "Participant")]
+        public async Task should_create_an_alert_when_the_quick_link_user_is_dismissed(Role role, string expectedPrefix)
+        {
+
+            var judge = TestConference.GetJudge();
+            var participant = TestConference.Participants.First(x => x.Role == role);
+            var user = new ClaimsPrincipalBuilder()
+                .WithUsername(judge.Username)
+                .WithRole(AppRoles.JudgeRole).Build();
+            Controller = SetupControllerWithClaims(user);
+
+            string expectedBody = $"{expectedPrefix} dismissed by {judge.HearingRole}";
+
+            var result = await Controller.DismissParticipantAsync(TestConference.Id, participant.Id);
+            var typedResult = (AcceptedResult)result;
+            typedResult.Should().NotBeNull();
+
+
+            VideoApiClientMock.Verify(x => x.AddTaskAsync(TestConference.Id,
+                It.Is<AddTaskRequest>(r => r.ParticipantId == participant.Id && r.Body == expectedBody && r.TaskType == TaskType.Participant)),
+                Times.Once);
+        }
+
         [Test]
         public async Task should_create_an_alert_when_the_witness_is_dismissed()
```

Comment:
As we are dismissing participants ( including QL paraticipants ) should we reflect this in the unit test name?